src/image_processing.rs

```diff
@@ -41,47 +45,81 @@ impl ImageTransformConfig {
             aspect_ratio_to_size.insert(aspect_ratio, *img_size);
         }
 
+        let mut aspect_ratios: Vec<(f64, String)> = aspect_ratio_to_size
+            .keys()
+            .filter_map(|k| k.parse::<f64>().ok().map(|f| (f, k.clone())))
+            .collect();
+        aspect_ratios.sort_by(|a, b| a.0.partial_cmp(&b.0).unwrap());
```

Comment:
micro optim, the aspect ratios are pre sorted now so we can find the appropriate one faster (this counts for almost nothing though..)